galite-core/src/main/kotlin/org/kopi/galite/report/PExport.kt

```diff
@@ -59,7 +59,7 @@ abstract class PExport(val table: UTable,
   }
 
   protected fun exportData() {
-    val group: VGroupRow = model.getTree()
+    val group: VGroupRow = model.root
```

Comment:
no

Reply:
This is the body of getTree() in MReport : 
fun getTree(): VGroupRow? {
    return root
  }
Else what's the difference ?

Reply:
I know, the implementation could change later